src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -47,7 +47,7 @@ object CromwellServer extends DefaultWorkflowManagerSystem {
   val webserviceConf = conf.getConfig("webservice")
 
   import scala.concurrent.ExecutionContext.Implicits.global
-  (IO(Http) ? Http.Bind(service, interface =  webserviceConf.getString("webservice.interface"), port = webserviceConf.getInt("webservice.port"))).onComplete {
+  (IO(Http) ? Http.Bind(service, interface =  webserviceConf.getString("interface"), port = webserviceConf.getInt("port"))).onComplete {
```

Comment:
could `.ask` and fix the messed up spacing while you're in here.  :stuck_out_tongue: